access_keys.go

```diff
@@ -0,0 +1,73 @@
+package cloudflare
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/pkg/errors"
+	"net/http"
+	"time"
+)
+
+type AccessKeysConfig struct {
+	KeyRotationIntervalDays int       `json:"key_rotation_interval_days"`
+	LastKeyRotationAt       time.Time `json:"last_key_rotation_at"`
+	DaysUntilNextRotation   int       `json:"days_until_next_rotation"`
```

Comment:
no `omitempty` or similar required on these JSON tags?

Reply:
(i think if you did, you'd be able to drop `AccessKeysConfigUpdateRequest` struct completely)

Reply:
None of these fields can ever be empty, would mean that something was wrong on the API side;

I added the `AccessKeysConfigUpdateRequest` with only one field exactly to prevent users from trying to update read-only fields, which I think its a better experience than letting the request go through only to return 400.

What do you think?

Reply:
for me, it's 6 of one and half a dozen of another. i don't think it's a big enough deal to warrant too much time but with some other resources, we've just made the tests/examples demonstrate reusing a shared structure. up to you what you want to do but i think it's fine either way.